app/experimenter/nimbus-ui/src/lib/mocks.tsx

```diff
@@ -30,55 +35,76 @@ export interface MockedState {
   client: ApolloClient<any>;
 }
 
-export const MOCK_CONFIG = {
+export const MOCK_CONFIG: ReturnType<typeof useConfig> = {
+  __typename: "NimbusConfigurationType",
+  graphql_url: "http://example.com/graphql",
+  sentry_dsn: "http://example.com/sentry",
+  version: "1.0",
```

Comment:
Hmm, maybe a non-issue, but I'm not sure if these values should live in this object. `MOCK_CONFIG` is loaded into the cache a little further down as the response for the `GET_CONFIG_QUERY` query. In practice these values would not come from this query, but instead would be merged in at the last moment in the `useConfig` hook. But then again, maybe this isn't a big deal as the values should be mocked anyway?

Reply:
Gave this a tweak - swapped over to type `getConfig_nimbusConfig` instead of the return val of `getConfig`